doc/index.rst

```diff
@@ -0,0 +1,66 @@
+.. include:: shields.inc
+
+.. image:: _static/logo.svg
+   :height: 90 px
+   :align: center
+   :target: https://GitHub.com/edaa-org/pyEDAA.ToolSetup
+
+|br|
+
+.. raw:: latex
+
+   \part{Introduction}
+
+.. only:: html
+
+   |  |SHIELD:svg:ToolSetup-github| |SHIELD:svg:ToolSetup-ghp-doc| |SHIELD:svg:ToolSetup-gitter|
+   |  |SHIELD:svg:ToolSetup-gha-test|
+
+.. Disabled shields: |SHIELD:svg:ToolSetup-src-license| |SHIELD:svg:ToolSetup-doc-license| |SHIELD:svg:ToolSetup-pypi-tag| |SHIELD:svg:ToolSetup-pypi-status| |SHIELD:svg:ToolSetup-pypi-python| |SHIELD:svg:ToolSetup-lib-status| |SHIELD:svg:ToolSetup-codacy-quality| |SHIELD:svg:ToolSetup-codacy-coverage| |SHIELD:svg:ToolSetup-codecov-coverage| |SHIELD:svg:ToolSetup-lib-dep| |SHIELD:svg:ToolSetup-req-status| |SHIELD:svg:ToolSetup-lib-rank|
+
+.. only:: latex
+
+   |SHIELD:png:ToolSetup-github| |SHIELD:png:ToolSetup-ghp-doc| |SHIELD:png:ToolSetup-gitter|
+   |SHIELD:png:ToolSetup-gha-test|
+
+.. Disabled shields: |SHIELD:png:ToolSetup-src-license| |SHIELD:png:ToolSetup-doc-license| |SHIELD:png:ToolSetup-pypi-tag| |SHIELD:png:ToolSetup-pypi-status| |SHIELD:png:ToolSetup-pypi-python| |SHIELD:png:ToolSetup-lib-status| |SHIELD:png:ToolSetup-codacy-quality| |SHIELD:png:ToolSetup-codacy-coverage| |SHIELD:png:ToolSetup-codecov-coverage| |SHIELD:png:ToolSetup-lib-dep| |SHIELD:png:ToolSetup-req-status| |SHIELD:png:ToolSetup-lib-rank|
+
+The pyEDAA.ToolSetup Documentation
+##################################
+
+*TBD*
+
+.. image:: _static/work-in-progress.png
+   :height: 275 px
+   :align: center
+   :target: https://GitHub.com/edaa-org/pyEDAA.ToolSetup
+
+|br|
+
+.. _contributors:
+
+Contributors
+************
+
+* `Patrick Lehmann <https://GitHub.com/Paebbels>`__ (Maintainer)
+* `Martin Zabel <https://GitHub.com/https://github.com/mzabeltud>`__
+* `Unai Martinez-Corral <https://GitHub.com/umarcor>`__
+* `and more... <https://GitHub.com/edaa-org/pyEDAA.ToolSetup/graphs/contributors>`__
+
+.. toctree::
+   :hidden:
+
+   Used as a layer of EDA² ➚ <https://edaa-org.github.io/>
+
+.. raw:: latex
+
+   \part{Appendix}
+
+.. toctree::
+   :caption: Appendix
+   :hidden:
+
+   genindex
```

Comment:
Add these sections:
* Link to coverage
* Link to static type check
* Documentation license

Reply:
Done.